torchao/dtypes/uintx/Uintx.py

```diff
@@ -11,10 +11,30 @@
     _dispatch__torch_dispatch__,
 )
 from torchao.dtypes.affine_quantized_tensor import PlainAQTLayout, register_layout_cls
-
+from torchao.utils import TORCH_VERSION_AFTER_2_3
 
 aten = torch.ops.aten
 
+# Note: Uintx does not work for torch 2.3 and below
+_DTYPE_TO_BIT_WIDTH = {}
+_BIT_WIDTH_TO_DTYPE = {}
+
+if TORCH_VERSION_AFTER_2_3:
+    _DTYPE_TO_BIT_WIDTH = {
```

Comment:
So this is the hacky fix until it is more natively supported

Reply:
right, `torch.uintx` is a placeholder right now and nothing is supported on it, we'd need to add a proper tensor subclass and connect it with these dtypes in the future